3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/errors.rb

```diff
@@ -29,6 +29,10 @@ class ConfigError < Error
     class EmrExecutionError < Error
     end
 
+    # Problem when discovering Amazon EMR JobFlow (e.g. multiple job-flows discovered with the same name)
+    class EmrDiscoveryError < Error
+    end
```

Comment:
We could do - TBH we should never hit that failure in practice as the Consul lock should prevent us from ever attempting to get a RUNNING cluster and we should never have a non-unique cluster.  If you think it would be useful happy to add one in.